R/qc_missed_cleavages.R

```diff
@@ -39,6 +39,7 @@ qc_missed_cleavages <-
     {
     result <- data %>%
       dplyr::distinct({{sample}}, {{grouping}}, {{missed_cleavages}}, {{intensity}}) %>%
+      tidyr::drop_na() %>%
```

Comment:
I would add the `remove_na_intensities` functionality as above also in this function. This makes it more consistent with other functions. Basically the user should by default be required to provide intensity so the function can remove NA intensities (because `remove_na_intensities` is by default TRUE).  Right now it is not clear that the function removes NA intensities if the user provides intensities while `method = "count"`. Just to make it transparent I would suggest these changes.